spec/features/user_accept_friend_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.feature 'User accept friend request' do
+  scenario 'log in and Send friend request' do
+    @user = FactoryBot.create(:user)
+    @other = FactoryBot.create(:user)
```

Comment:
`[Optional]` Favor the use of `let` instead of instance variables.

Reply:
I don't understand what you mean about let, that's a javascript thing right?